apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff

```diff
@@ -0,0 +1,796 @@
+      </trans-unit>
+      <trans-unit id="If you received SimpleX Chat invitation link you can open it in your browser:" xml:space="preserve">
+        <source>If you received SimpleX Chat invitation link you can open it in your browser:</source>
+        <target>Если Вы получили ссылку с приглашением из SimpleX Chat, Вы можете открыть её в браузере:</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Install [SimpleX Chat for terminal](https://github.com/simplex-chat/simplex-chat)" xml:space="preserve">
+        <source>Install [SimpleX Chat for terminal](https://github.com/simplex-chat/simplex-chat)</source>
+        <target>[SimpleX Chat для терминала](https://github.com/simplex-chat/simplex-chat)</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Invalid connection link" xml:space="preserve">
+        <source>Invalid connection link</source>
+        <target>Ошибка в ссылке контакта</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Make sure SMP server addresses are in correct format, line separated and are not duplicated." xml:space="preserve">
+        <source>Make sure SMP server addresses are in correct format, line separated and are not duplicated.</source>
+        <target>Пожалуйста, проверьте, что адреса SMP серверов имеют правильный формат, каждый адрес на отдельной строке и не повторяется.</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Markdown in messages" xml:space="preserve">
+        <source>Markdown in messages</source>
+        <target>Форматирование сообщений</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Message delivery error" xml:space="preserve">
+        <source>Message delivery error</source>
+        <target>Ошибка доставки сообщения</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Most likely this contact has deleted the connection with you." xml:space="preserve">
+        <source>Most likely this contact has deleted the connection with you.</source>
+        <target>Скорее всего, этот контакт удалил соединение с вами.</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="New contact request" xml:space="preserve">
+        <source>New contact request</source>
+        <target>Новый запрос на соединение</target>
+        <note>notification</note>
+      </trans-unit>
+      <trans-unit id="New message" xml:space="preserve">
+        <source>New message</source>
+        <target>Новое сообщение</target>
+        <note>notifications</note>
+      </trans-unit>
+      <trans-unit id="Notification are disabled!" xml:space="preserve">
+        <source>Notification are disabled!</source>
```

Comment:
seems we have a typo in key here, probably easier to fix while there's no localizations